composer.json

```diff
@@ -17,11 +17,13 @@
     "require": {
         "php":                           ">=5.4",
         "behat/mink-browserkit-driver":  "~1.2@dev",
-        "fabpot/goutte":                 "~1.0.4|~2.0|~3.1"
+        "fabpot/goutte":                 "~1.0.4|~2.0|~3.1|^4.0"
     },
 
     "require-dev": {
-        "mink/driver-testsuite": "dev-master"
+        "mink/driver-testsuite": "dev-master",
+        "phpunit/phpunit": "^9.0",
+        "guzzlehttp/guzzle": "^7.0"
```

Comment:
This guzzle requirement does not make any sense, because Goutte 4 is not based on guzzle at all anymore anyway.